### ISSUE TITLE
Changes from pull request #121

### ISSUE THREAD

**Issue by emaks** [yandex-qatools/htmlelements#122]
Changes from pull request #121 have removed difference between ```TypifiedElement.class``` and ```HtmlElement.class```. Now we can do all actions that provides selenium api using ```TypifiedElement.class``` while this class has been implemented not to provide this possibility

**Comment by ham1** [comment]
This is correct. What do you see the disadvantages of being able to access the `WebElement` API with `TypifiedElement`s? Do you know why it was different in the first place?

I found the major advantage being able to wait for it like a normal `WebElement`.
It also reduces code duplication, often you want to perform actions which are identical to the `WebElement` API and before the PR they were been passed through, without modification, to the wrapped `WebElement`.

**Comment by ham1** [comment]
Why *must* the functionality not be there?

Is already is there as you can still get access to the API using `.getWrappedElement()` - so easy to get around.

You can chose just not to use it.

Whereas, if it's not there, `.getWrappedElement()` has to be called or things such as explicit "wait for (in)visibility of all in" a `List` *cannot* be done without a custom `ExpectedConditions`.

In your example, you might want `getText()` on a button to check it has the expected text, also you might want to send the enter key on the button to test keyboard navigation.

**Comment by emaks** [comment]
> Why must the functionality not be there?

because it looks like https://hsto.org/getpro/habr/post_images/9b8/71b/a0b/9b871ba0b4fe575aa9d896a6d642c55a.jpg

**Comment by lanwen** [comment]
`TypifiedElements` is a architect fail. So in most cases we have more troubles than profit from using it. So  its just a step to make unification of this classes

**Comment by emaks** [comment]
in that case ```TypifiedElements``` have to be removed

**Comment by ham1** [comment]
I totally agree. As well as being more 'correct' it will simplify the code base, which is always good, but what of backward compatibility?

**Comment by lanwen** [comment]
yep, we can't just remove this class without set of small steps which are: unification - deprecation - removing.

**Comment by ham1** [comment]
Can this be closed or are we going to look to deprecate `TypifiedElement`s?

**Comment by artkoshelev** [comment]
i would vote for deleting it
